app/models/resource.rb

```diff
@@ -8,14 +11,26 @@ class Resource < ActiveRecord::Base
 
   # Validations
   validates_presence_of :title, :paradigm_id
+  validates_numericality_of :paradigm_id, only_integer: true
 
   # Scopes
   scope :alphabetical, -> { order("title") }
   scope :active, -> { where('active = ?', true) }
   scope :inactive, -> { where('active = ?', false) }
 
-
   # Methods
+  private
+    # Ensure that all links are saved with 'http://'
+    # so that rails doesn't think the external links are relative links
+    def reformat_link
+      reg = /^https?:\/\//
+      link = self.link
+      if !link.match(reg)
+        link.insert(0,"http://")
+      end
+      self.link = link
+    end
+
   def self.parse(spreadsheet, paradigms)
```

Comment:
This method can't be private, or else the imports controller won't be able to call this method. Just put it above the "private" line. 
Also did the imports controller tests pass? Cuse it should have thrown an error